qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/JavadocEmptyLineCheck/Valid.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * This is not a real Java class. It won't be compiled ever. It is used
+ * only as a text resource in integration.ChecksIT.
+ */
+public final class Valid {
+
+    /**
+     * There is no empty Javadoc line at the beginning.
+     */
+    private static final int x = 0;
+
+    /**
+     * There is no empty Javadoc line at the beginning.
+     */
+    private int y = 0;
+
+    /**
+     * There is no empty Javadoc line at the end.
+     */
+    private int z = 0;
+
+    /**
+     * There is no empty Javadoc line at the beginning.
+     */
+    public Valid() {}
+
+    /**
+     * There is no empty Javadoc line
+     * @param param - param.
+     * @return Param.
+     */
+    public String method(final String param) {
+        return param;
+    }
+
+    /*
+     *
+     * It's not Javadoc, just some comment
+     */
+    public void method2() {
+    }
+
+    /**
+     * First.
+     *
+     * Second.
+     */
+    public void method3() {
+    }
+
+    /** Some text
+     *
```

Comment:
@mkordas if it's valid Javadoc JavadocEmptyLineCheck should prohibit because there is empty line at the end.